jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/MultiPartConfig.java

```diff
@@ -0,0 +1,106 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.server;
+
+import java.nio.file.Path;
+
+import org.eclipse.jetty.http.ComplianceViolation;
+import org.eclipse.jetty.http.MultiPartCompliance;
+
+import static org.eclipse.jetty.http.ComplianceViolation.Listener.NOOP;
+
+public class MultiPartConfig
```

Comment:
What's the scope of this class?
Seems like a duplicate of the parser configuration, and it cannot be used on the client.